persistence-elasticsearch/core/src/main/resources/META-INF/cxs/mappings/rule.json

```diff
@@ -42,6 +42,9 @@
     },
     "raiseEventOnlyOnceForSession": {
       "type": "boolean"
+    },
```

Comment:
I'm not sure how this modification would impact existing rule sets. Could you test this before and after adding this definition? If it fails this means a migration using the Migration commands will need to be implemented for this change.